sdc/tests/test_dataframe.py

```diff
@@ -1036,12 +1036,44 @@ def test_impl(df):
         h_out = hpat_func(df)
         pd.testing.assert_frame_equal(out, h_out)
 
-    @skip_numba_jit
-    def test_df_drop1(self):
+    def test_df_drop_one_column(self):
         def test_impl(df):
-            return df.drop(columns=['A'])
+            return df.drop(columns='A')
 
-        df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0], 'B': [4, 5, 6, 7]})
+        df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0], 'B': [4, 5, 6, 7], 'C': [1.0, 2.0, np.nan, 1.0]})
+        hpat_func = self.jit(test_impl)
+        pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
+
+    def test_df_drop_tuple_column(self):
+        def test_impl(df):
+            return df.drop(columns=['A', 'B'])
+
+        # Numba supports only tuple iteration
+        def test_sdc_impl(df):
+            return df.drop(columns=('A', 'B'))
```

Comment:
Let's remove `test_impl` where `columns` is a list and test the feature with only `columns` as a tuple, because of the Numba limitation. Let's apply that for all the tests.